src/main/java/de/tum/in/www1/artemis/service/AssessmentDashboardService.java

```diff
@@ -107,7 +132,59 @@ public void generateStatisticsForExercisesForAssessmentDashboard(Set<Exercise> e
                     });
             exercise.setTutorParticipations(Collections.singleton(tutorParticipation));
 
-            log.debug("StatsTimeLog: tutor participations done in " + TimeLogUtil.formatDurationFrom(start) + " for exercise " + exercise.getId());
+            log.info("Finished >> assessmentDashboardLoopIteration << call for exercise " + exercise.getId() + " in " + TimeLogUtil.formatDurationFrom(start2));
+        }
+        log.info("Finished >> generateStatisticsForExercisesForAssessmentDashboard << call in " + TimeLogUtil.formatDurationFrom(startComplete));
+    }
+
+    /// **
+    // * This method fetches and stores the number of assessments for each exercise.
+    // * // TODO: implement
```

Comment:
This TODO here too?

Reply:
I removed this whole part and left one comment in line 77 that we could implement this part.